modules/core/src/smithy4s/internals/DocumentEncoderSchemaVisitor.scala

```diff
@@ -96,10 +96,6 @@ class DocumentEncoderSchemaVisitor(
   def this(cache: CompilationCache[DocumentEncoder]) =
     this(cache, explicitDefaultsEncoding = false)
 
-  @deprecated
```

Comment:
Just noting this didn't have a message or version in case that warrants scrutiny.